apps/agentfabric/user_core.py

```diff
@@ -18,9 +18,13 @@
 def init_user_chatbot_agent(uuid_str=''):
     builder_cfg, model_cfg, tool_cfg, available_tool_list = parse_configuration(
         uuid_str)
+    # set top_p and stop_words for role play
+    model_cfg[builder_cfg.model]['generate_cfg']['top_p'] = 0.5
+    model_cfg[builder_cfg.model]['generate_cfg']['stop'] = 'Observation'
```

Comment:
is it necessary to set it in mode_config.json as the default config?

Reply:
Due to our desire to use the default configuration top_p=0.8 in other situations, and this setting is only applicable to role-play, so I written it separately here.

Reply:
what about the stop parameter?  I think it is necessary for other qwen series